openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -103,12 +102,12 @@ <h4 class="question__title ui-slidable__control">
 
                     {% if peer_assessments %}
                         <form class="submission__feedback submission__feedback step__content__section ui-slidable__container is--showing" method="post">
-                            <h3 class="submission__feedback__title ui-slidable__control">
+                            <h4 class="submission__feedback__title ui-slidable__control">
```

Comment:
I don't see how it can move to an h4 though because h4 is used for the step title (see line 5).